src/components/icon-tile/style.module.css

```diff
@@ -52,6 +52,17 @@
   --background: var(--token-color-neutral-background-secondary);
 }
 
+/*
+Note: color-neutral-dark is not part of IconTile's official HDS spec,
+but is necessary for us to implement the current designs for the 
+/sentinel and /hcp product landing pages.
+*/
+.color-neutral-dark {
```

Comment:
As noted in comments, need this net new `IconTile` theme to implement the `/sentinel` and `/hcp` product landing pages.